google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -149,7 +154,7 @@ private static <V> V get(ApiFuture<V> future) {
       return Uninterruptibles.getUninterruptibly(future);
     } catch (ExecutionException ex) {
       Throwables.throwIfUnchecked(ex.getCause());
-      throw new RuntimeException(ex);
+      throw new VerifyException(ex);
```

Comment:
I believe it might impact the existing contract with our library users - if user expecting `RuntimeException`, it will break

Reply:
`VerifyException` is a child class of `RuntimeException`. If they have a `catch RuntimeException` this will still work.

I can imagine a circumstance in which they're checking for the exact class using getClass but I think that would be a really weird thing to do.

Reply:
OK, I believe we can take this risk then